src/modules/alerting.ts

```diff
@@ -126,35 +144,31 @@ class AlertingApi extends BaseApi {
     >(
         title: string,
         message: string,
-        options: ToastOptions<MessageType, Timeout> = {
-            type: 'info',
-            timeout: 7,
-            showCloseButton: true,
-            markdownMessage: true
-        } as unknown as ToastOptions<MessageType, Timeout>
```

Comment:
I'm not sure I understand the bug here.  is it a runtime bug or a bug in generating the docs?

it's true that the type assertion to unknown first is risky but it's possibly acceptable here since the default is a constant and we're not taking any liberties with user provided values.

the default should only be applied if options is undefined, so, while

```ts
myClient.alerting.showToastPopup<'info', number>('foo', 'bar');
```

is technically incorrect, the function won't hold the type params so it's not as bad as a typed class which uses the types in other aspects of the class.

This does erorr correctly:
```ts
api.showToastPopup<'info', number>('foo', 'bar', {
    type: 'error',
    showCloseButton: true,
    timeout: 0
});

And the function types can be inferred from the final argument

```ts
api.showToastPopup('foo', 'bar', {
    type: 'error',
    showCloseButton: true,
    timeout: 0
});
```

I'm not opposed to this change at all, just not understanding the bug aspect of it.  I'm also not sure if applying the default inside the function is any more correct except that it does remove the type assertions on the default param.  But, it's very possible I'm missing a nuance.

Reply:
ah, i see.  seems like we're not escaping the values going into the md table and that's what causes the issue.  specifically the `|` char in the union type is probably doing it.  I wonder if there is something we could change in the doc generation so we wouldn't have to change the code?

good to know the issue isn't at runtime or with the types

Reply:
There is no `|` in the parameters, it's looking like it's a whitespace issue instead. When I get rid of the new character lines in the options parameter it does format the table correctly, although it does keep the type assertions which don't feel like they belong in the docs. 
![image](https://github.com/MyPureCloud/client-app-sdk/assets/51763164/7835a000-f12e-43ac-9411-46b288d80a9c)
I'm not too sure there's much we could do about the whitespace in terms of doc generation but I can take a look into it if we want to go that path. Thoughts?